src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class OperationAnalyzerTests : CompilingTestBase
     {
+        private static CSharpParseOptions patternParseOptions =
+            TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp6)
+                    .WithFeature("patterns", "true")
+                    .WithFeature("patternsExperimental", "true");
```

Comment:
Yes, that would be a great idea for a global refactoring of the way we handle features. I'd prefer not to mix that with these changes.